File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/GuitarristaFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Guitarrista;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class GuitarristaFactoryTest {

    @Test
    void createGuitarristaTest() {
        Guitarrista Guitarrista = GuitarristaFactory.create();
        // Guitarrista no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(Guitarrista);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/GuitarristaFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class GuitarristaFactoryTest {
    @Test
    fun createGuitarristaTest() {
        val Guitarrista: Musico.Guitarrista = GuitarristaFactory.create()
        // Guitarrista no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(Guitarrista)
    }
}